examples/docs_snippets/docs_snippets/intro_tutorial/advanced/materializations/materializations.py

```diff
@@ -34,26 +32,22 @@ def sort_by_calories(context, cereals):
     logger.info(f"Most caloric cereal: {most_caloric}")
 
     fieldnames = list(sorted_cereals[0].keys())
-    sorted_cereals_csv_path = os.path.abspath(
-        f"output/calories_sorted_{context.run_id}.csv"
-    )
+    sorted_cereals_csv_path = os.path.abspath(f"output/calories_sorted_{context.run_id}.csv")
     os.makedirs(os.path.dirname(sorted_cereals_csv_path), exist_ok=True)
 
     with open(sorted_cereals_csv_path, "w") as fd:
         writer = csv.DictWriter(fd, fieldnames)
         writer.writeheader()
         writer.writerows(sorted_cereals)
 
-    yield AssetMaterialization(
```

Comment:
Worth checking to see if the text of the corresponding tutorial page needs any updates.